benchmarks/benchmark_shared.c

```diff
@@ -42,6 +42,9 @@
 #include "shared.h"
 #include "benchmark_shared.h"
 
+#define FT_MIN_ITERS 10
+#define FT_BW_ITERS MAX(opts.iterations / MAX(opts.window_size / 8, 1), FT_MIN_ITERS)
```

Comment:
If we just use opts.iterations / opts.window_size, the default window size would result in just one iteration for the largest message size that we use and any window size higher than that would result in zero iterations. We can define the FT_MIN_ITERS to 5 to match with the quick test.